src/vampy/__init__.py

```diff
@@ -8,21 +8,27 @@
 from .automatedPostprocessing import postprocessing_common
 from .automatedPostprocessing import visualize_probes
 # Imports from pre-processing
-from .automatedPreprocessing import DisplayData
-from .automatedPreprocessing import ImportData
-from .automatedPreprocessing import NetworkBoundaryConditions
-from .automatedPreprocessing import ToolRepairSTL
-from .automatedPreprocessing import automated_preprocessing
-from .automatedPreprocessing import preprocessing_common
-from .automatedPreprocessing import simulate
-from .automatedPreprocessing import visualize
-from .automatedPreprocessing import vmtk_pointselector
+try :
```

Comment:
I think there should be no white space between `try`and `:` (colon). The same goes for `except` below.